src/flick/user/serializers.py

```diff
@@ -61,3 +63,39 @@ class Meta:
             "collab_lsts",
         )
         read_only_fields = fields
+
+
+class FriendProfileSerializer(serializers.ModelSerializer):
+    profile_id = serializers.CharField(source="id")
+    user_id = serializers.CharField(source="user.id")
```

Comment:
haha I recently made a change that got rid of this--instead of having a `profile_id` and a `user_id`, we only need `id` (`id` of `User`). so you should replace these two lines with `id = serializers.IntegerField(source="user.id")`